cmd/dreamboat/main.go

```diff
@@ -219,6 +224,24 @@ var flags = []cli.Flag{
 		Value:   2,
 		EnvVars: []string{"RELAY_SUBMISSION_LIMIT_BURST"},
 	},
+	&cli.StringFlag{
```

Comment:
I made it pretty simple, like without any preoptimisation. 
Assuming that:
* ipc is local anyway, so it wouldn't make this much sense. 
* I assumed that as http are separate network connections it would be under some kind of LB anyway. 
* ws acctually allows you to add more than one connection to round robin, but for simplicity I'm not reusing the same connection type, just fallback to the slower one (http). We can add this later but not sure if that should be the priority right now